libsplinter/src/rest_api/auth/authorization/permission_map.rs

```diff
@@ -56,6 +62,39 @@ impl PermissionMap {
     }
 }
 
+#[derive(PartialEq, Clone)]
+pub enum Method {
+    Get,
+    Post,
+    Put,
+    Patch,
+    Delete,
+    Head,
+    Options,
+    Connect,
+    Trace,
+    Extension(String),
+}
+
+impl From<&Actix1Method> for Method {
```

Comment:
Will this move to the actix-1 crate (in a future PR)?

Reply:
Yeah, it probably won't look exactly like this because of the crate boundaries but the essence yes